src/app/app-routing.module.ts

```diff
@@ -18,6 +19,8 @@ const routes: Routes = [
     path: 'signIn', component: SignInComponent
   }, {
     path: 'dashboard', component: DashboardComponent
+  }, {
+    path: 'newTrip', component: NewTripComponent
```

Comment:
This is a dialog, I think you do not need this in your routes. There is no path /newTrip in your application. The dialog will just be displayed in your dashboard route.